README.md

```diff
@@ -0,0 +1,36 @@
+# Minetest Game
+
+The default game bundled in the Minetest engine.  
+
+For further information, check [this forum topic](https://forum.minetest.net/viewtopic.php?f=15&t=9724).   
+Also see the [Minetest Wiki](https://wiki.minetest.net/Subgames/Minetest_Game) for more information.
+
+## Installation
+
+- Unzip the archive, rename the folder to minetest_game and
+place it in .. minetest/games/
+
+- GNU/Linux: If you use a system-wide installation place
+    it in ~/.minetest/games/.
+
+The Minetest engine can be found at
+	[GitHub](https://github.com/minetest/minetest).
+
+For further information or help, see:  
+https://wiki.minetest.net/Installing_Mods
+
+## Compatibility
+
+The Minetest Game github master HEAD is generally compatible with the github
+master HEAD of the Minetest engine.  
+Additionally, when the Minetest engine is tagged to be a certain version (eg.
+0.4.10), Minetest Game is tagged with the version too.
+
+When stable releases are made, Minetest Game is packaged and made available in
+	https://minetest.net/downloads/
+and in case the repository has grown too much, it may be reset. In that sense,
+this is not a "real" git repository (Package maintainers please note).
```

Comment:
Would this actually ever be done? It seems like a bad idea and completely unnecessary.

Reply:
It **does** sounds like a bad idea, though.
I will keep it there for now. I'm not really sure if that should be deleted anyway.

Reply:
```
and in case the repository has grown too much, it may be reset. In that sense,
this is not a "real" git repository. (Package maintainers please note!)
```
Oh actually, thanks for spotting this, it dates from MTG 0.4.0, 7 years ago and i'm fairly sure it is no longer correct, what it states is bizarre. Please remove.

Reply:
Done.